### PR TITLE
Fixed vice.com

### PR DESCRIPTION
**Pull request by sprinkledcheese**
- Inverted VICE NEWS logo.
- Made VICE logo visible.

How can I make dark reader not apply on the header ?

### CODE REVIEW

File: src/config/dynamic-theme-fixes.config [hunk 1]
```diff
@@ -4026,6 +4026,18 @@ CSS
 
 ================================
 
+vice.com
+
+INVERT
+img[src*="1565710323798-article-logo-news.svg"]
+
+CSS
+div.titlebar__logo.dsp-flex--xs.grd-algn-center-xs a div svg path {
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:4035]:
```
IGNORE INLINE STYLE
.titlebar__logo__icon svg path
```
Better approach?

File: src/config/dynamic-theme-fixes.config [hunk 2]
```diff
@@ -4026,6 +4026,18 @@ CSS
 
 ================================
 
+vice.com
+
+INVERT
+img[src*="1565710323798-article-logo-news.svg"]
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:4032]: Unable to find this CSS selector.

File: src/config/dynamic-theme-fixes.config (hunk 1 above)
Comment (sprinkledcheese) [reply src/config/dynamic-theme-fixes.config]: Done.

File: src/config/dynamic-theme-fixes.config (hunk 2 above)
Comment (sprinkledcheese) [reply src/config/dynamic-theme-fixes.config:4032]: https://www.vice.com/en_us/article/g5pqk4/german-neo-nazis-are-getting-explosives-training-at-a-white-supremacist-camp-in-russia

logo above the headline.

Comment (Gusted) [reply src/config/dynamic-theme-fixes.config:4032]: Remove `1565710323798-` as this is generated per article.

Comment (Gusted) [inline src/config/dynamic-theme-fixes.config]: Remove `1565710323798-`

File: src/config/dynamic-theme-fixes.config [hunk 3]
```diff
@@ -4026,6 +4026,18 @@ CSS
...
+
+CSS
+.titlebar__logo__icon svg path {
+    --darkreader-inline-fill: ${black} !important;
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:4036]: Remove
```
{
    --darkreader-inline-fill: ${black} !important;
}
```
And change CSS to IGNORE INLINE STYLE

also add `.logo-vice__foreground` For this sites https://www.vice.com/en_us/section/tech

**Comment by sprinkledcheese** [comment]
Done.

File: src/config/dynamic-theme-fixes.config (hunk 3 above)
Comment (sprinkledcheese) [reply src/config/dynamic-theme-fixes.config:4038]: Also fixed hamburger menu icon and footer icon. The cards are still white on the main page.

**Pull request merged**